src/store/sagas/positions.ts

```diff
@@ -47,8 +48,19 @@ function* handleInitPositionAndPoolWithSOL(action: PayloadAction<InitPositionDat
   ) {
     return yield* call(handleInitPosition, action)
   }
-
+  const loaderCreatePool = (new Date().getMilliseconds() + Math.random()).toString()
+  const loaderSigningTx = (new Date().getMilliseconds() + Math.random()).toString()
```

Comment:
The method of determining the `key` can also be encapsulated in a utility function, making it easier to control it in one place. This formula, although short, is repeated multiple times

Reply:
that's true, i've created createLoaderKey util for it.